fpga/src/bootrom/linker.lds

```diff
@@ -0,0 +1,38 @@
+ENTRY(main)
+
+SECTIONS
+{
+    ROM_BASE = 0x83000000; /* ... but actually position independent */
```

Comment:
I think this `ROM_BASE` will make problems if the boot-vector actually point to `0x10000`. Right?

Reply:
Ups! Fixed